test/asBuffer.js

```diff
@@ -0,0 +1,40 @@
+const test = require('tape')
+const fixtures = require('./fixtures')
+const pdf = require('../lib')
+
+const f = fixtures.create();
+
+let doc = new pdf.Document({
+  font:       f.font.afm.regular,
+  padding:    10,
+  lineHeight: 1,
+  info: {
+    id = '42',
+    creationDate = new Date(2015, 1, 19, 22, 33, 26),
+    producer = 'pdfjs tests (github.com/rkusa/pdfjs)'
```

Comment:
s/ = / : / 😃

Reply:
Yep, I have just seen that, sorry.
I had to delete and re-clone the repo to finally successfully launch all the tests (I previously have an XRef error on external docs) but now I can check that the test is ok before pushing the commit. :p

Reply:
And I have talked too fast...
I still have an issue on the reading of `pdfjs-created.pdf` for the fixtures.
But the asBuffer test should run.